build.py

```diff
@@ -593,6 +593,7 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap, backends):
             unzip \
             wget \
             zlib1g-dev \
+            doxygen \
```

Comment:
Isn't doxygen only used for generating documentation?

Reply:
Yes, I think it may be an optional decency, I will remove this